docs/content/developers/network-test-environments.md

```diff
@@ -0,0 +1,226 @@
+---
+search:
+  boost: .1
+---
+# Network Test Environments
+
+## Packet-Inspection VPN Simulation
```

Comment:
I think headings should be changed on this page (i.e., remove one `#` from each), because Table of contents shows only one item:

![image](https://github.com/user-attachments/assets/897e0414-dfdf-45b0-a54f-af08f7470698)